packages/zoe/src/contracts/atomicSwap.js

```diff
@@ -1,45 +1,29 @@
-/* eslint-disable no-use-before-define */
 import harden from '@agoric/harden';
 
 // Eventually will be importable from '@agoric/zoe-contract-support'
 import { makeZoeHelpers } from '../contractSupport';
 
 // zcf is the Zoe Contract Facet, i.e. the contract-facing API of Zoe
 export const makeContract = harden(zcf => {
-  const { swap, assertKeywords, rejectIfNotProposal } = makeZoeHelpers(zcf);
-  assertKeywords(harden(['Asset', 'Price']));
```

Comment:
If the keywords were computed or carried from elsewhere by a variable, I would understand the purpose of this line. But here they are literal so you already know they're valid. And if you made a mistake, Zoe would catch it anyway.

Let's talk about this one.

Reply:
I think you're misunderstanding what this function does, or I'm misunderstanding what you're saying here. This function is getting the issuerKeywordRecord for the contract from Zoe, and asserting that the actual keywords match the expected keywords (here: ['Asset', 'Price']). This is an important task, because the user might put any keywords in the issuerKeywordRecord, and this is the only check. 

I think, based on your comment, that you are thinking that this function is checking whether the string 'Asset' and the string 'Price' are valid keywords. This function is not doing that.

Let me know if I've misunderstood you, but this line as I understand it should stay in.

Reply:
Thanks! Yes, I had exactly this misunderstanding. I now remember we'd discussed it before but I misremembered.

Will restore these.